src/main/plugin/iso19139.ca.HNAP/update-fixed-info.xsl

```diff
@@ -955,6 +955,51 @@
     </xsl:copy>
   </xsl:template>
 
+  <!-- Fix the dates for the Core Subject Thesaurus -->
```

Comment:
This seems to be a bug specifically to fix "Core Subject Thesaurus" Thesarurus

I would have expected a fix that would not change the Thesarurus date if a value already exists.  Otherwise are we not going to have the same issue with the other thesaurus?